test/sql/append.sql

```diff
@@ -0,0 +1,110 @@
+\ir include/create_single_db.sql
+
+-- create a now() function for repeatable testing that always returns
```

Comment:
I don't want to put all queries in one big query tests, so I opted to keep it modular by making a similar diff for the append tests. I think this should be the approach going forward.